teuthology/openstack/__init__.py

```diff
@@ -1075,7 +1075,12 @@ def get_user_data(self):
                 "{user} >> /tmp/init.out "
                 "2>&1".format(user=self.username,
                               opts=' '.join(setup_options + all_options))),
-            "/etc/init.d/teuthology restart"
+                # wa: we want to stop paddles and pulpito started by
+                # setup-openstack before starting teuthology service
+                "pkill -f 'pecan serve'",
+                "pkill -f 'python run.py'",
+                "systemctl enable teuthology",
+                "systemctl start teuthology",
```

Comment:
I don't understand the indentation here. The patch seems to be removing one entry from the `cmds` list and adding four new entries in its place. If so, the four new entries should be moved back one level of indentation?

Reply:
Though the indentation does not play role in this particular case only cosmetic, you're right it should be one level upper.